README.md

```diff
@@ -18,6 +18,11 @@ Maintenance Status: Stable
 
 - [Getting Started](#getting-started)
 - [Documentation](#documentation)
+  - [API](#api)
+    - [`player.overlay()`](#playeroverlay)
+    - [`overlay.overlays`](#overlayoverlays)
```

Comment:
Maybe this should be a getter, ex. `getOverlays()`, rather than a property?

Reply:
I would suggest calling it `get()` to align with the other new methods.